prow/tide/status_test.go

```diff
@@ -420,6 +446,7 @@ func TestExpectedStatus(t *testing.T) {
 					IncludedBranches: tc.branchWhitelist,
 					Labels:           neededLabels,
 					MissingLabels:    forbiddenLabels,
+					Author:           "batman",
```

Comment:
Please test the behavior when only one of the two queries specifies an author and the PR author does not match.